.travis.yml

```diff
@@ -21,8 +21,15 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+services:
+  - docker
+
+before_install:
+  - docker pull ibmcom/ibmjava:8-sdk
```

Comment:
Hi @chtompki , I just [compared](https://www.diffchecker.com/y4s7vOx6) them and found both Dockerfile belongs to same repository, I took the one from master. Official belongs to some past commit. I prefer to go with official ``` library/ibmjava:8-sdk ``` though.